cpp/include/Ice/SSL/ConnectionInfo.h

```diff
@@ -22,6 +26,12 @@ namespace Ice::SSL
 #if defined(ICE_USE_SCHANNEL)
     /**
      * Provides access to the connection details of an SSL connection.
+     *
+     * The SchannelConnectionInfo class is only available when the %Ice library is built on Windows. For Linux,
+     * refer to OpenSSLConnectionInfo, and for macOS and iOS, refer to SecureTransportConnectionInfo.
+     *
+     * Additionally, the `ConnectionInfo` alias is defined for use in portable code, representing the
+     * platform-specific connection info class.
```

Comment:
I added this comment here, and in other platform dependent classes to clarify the platforms for each class. I also mentioned the alias for portable code, as it is not documented otherwise.